routers/object.js

```diff
@@ -403,6 +403,22 @@ router.post('/:objectName/frame/:frameName/pinned/', function (req, res) {
     });
 });
 
+router.post('/:objectName/requestGaussianSplatting/', async function (req, res) {
+    if (!utilities.isValidId(req.params.objectName)) {
+        res.status(400).send('Invalid object name. Must be alphanumeric.');
+        return;
+    }
+    // splat status (commonly referred to as "splattus") is
+    // {done: boolean, progress: number}
```

Comment:
Is the "splattus" (great name btw):
`{done: boolean, progress: number}` or `{done: boolean, gaussianSplatRequestId: string}`?

Reply:
Oh, sorry! Will update accordingly